jenkins/check_projects.py

```diff
@@ -258,9 +258,9 @@ def load_projects(configs, boskos, filt):
                             raise ValueError(project)
 
         with open(boskos) as fp:
-            for res in json.loads(fp.read()):
-                if res['type'] == 'project':
-                    projects.add(res['name'])
+            for rtype in json.loads(fp.read()):
+                for name in rtype['names']:
```

Comment:
maybe add `if rtype['type'] == 'project':` to future-proof this if other resource-types are added?